docs/plugins/analysis-icu.asciidoc

```diff
@@ -413,6 +413,11 @@ The following parameters are accepted by `icu_collation_keyword` fields:
     Accepts a string value which is substituted for any explicit `null`
     values.  Defaults to `null`, which means the field is treated as missing.
 
+`ignore_above`::
```

Comment:
A couple small comments:
- For the heading, we can do `<<ignore-above,`ignore_above`>>::` as we do in `keyword.asciidoc`. This will provide a link to the main documentation for `ignore_above`.
- It would be good to add information about the default behavior (something like "by default, there is no limit and all values will be indexed").T